src/components/common/Drawer.jsx

```diff
@@ -0,0 +1,45 @@
+import { Drawer, Box, Typography, IconButton, Icon, Menu } from "@mui/material";
+import MenuIcon from "@mui/icons-material/Menu";
+import ArrowBackIosNewSharpIcon from "@mui/icons-material/ArrowBackIosNewSharp";
+import { styled } from "@mui/material/styles";
+
+import { useState } from "react";
+
+const DrawerHeader = styled("div")(() => ({
+  display: "flex",
+  alignItems: "center",
+  justifyContent: "flex-end",
+}));
+
+export const MuiDrawer = () => {
```

Comment:
Did you use somewhere the Drawer component? or is it going to be used in a future branch?
Also change the name into Drawer. You will also have to use an alias on line 1 where you imported the Drawer. 
`import { Drawer as MuiDrawer, ... } from ...`

Reply:
found the location where was used. It has to be moved from there.